docs/intro/install.rst

```diff
@@ -95,10 +95,6 @@ Ubuntu 9.10 or above
 **Don't** use the ``python-scrapy`` package provided by Ubuntu, they are
 typically too old and slow to catch up with latest Scrapy.
 
-Instead, use the official :ref:`Ubuntu Packages <topics-ubuntu>`, which already
-solve all dependencies for you and are continuously updated with the latest bug
-fixes.
-
 If you prefer to build the python dependencies locally instead of relying on
```

Comment:
With the removal of the recommendation for Ubuntu packages, _"If you prefer to build the python dependencies locally"_ sounds a bit weird since the dependencies are now not mentioned earlier.
I believe they should be listed at a minimum (`twisted`, `cryptography`, `lxml` may be sufficient)

A link to their respective install pages for ubuntu would also help users when troubleshooting, and shed light on the list of non-python dependencies listed below.